kaocha-runner.el

```diff
@@ -46,6 +46,18 @@
   :group 'kaocha-runner
   :type 'string)
 
+(defcustom kaocha-runner-output-win-height
+  12
+  "The output window height."
+  :group 'kaocha-runner
+  :type 'integer)
```

Comment:
It's also a good idea to add a `:package-version` properties to defcustoms.

Reply:
Thank you, added the package version.